common/changes/@microsoft/rush/master_2019-11-20-02-32.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Removing registry check from --pack, since --pack might need to run view against a registry to check if a package exists",
+      "type": "patch"
```

Rush is versioned with a version policy, so this should stay empty.

```suggestion
      "type": ""
```